src/Sylius/Bundle/CoreBundle/Kernel/Kernel.php

```diff
@@ -173,4 +174,16 @@ protected function isVagrantEnvironment()
     {
         return (getenv('HOME') === '/home/vagrant' || getenv('VAGRANT') === 'VAGRANT') && is_dir('/dev/shm');
     }
+
+    /**
+     * @return string
+     */
+    protected function getContainerBaseClass()
+    {
+        if ('test' === $this->environment || 'test_cached' === $this->environment) {
+            return MockerContainer::class;
```

Comment:
Why only to test environment ? If i am correct travis running on test_cached, so mockery container won't work.